open-api/rest-catalog-open-api.yaml

```diff
@@ -3138,6 +3149,8 @@ components:
           description: May be null if the table is staged as part of a transaction
         metadata:
           $ref: '#/components/schemas/TableMetadata'
+        planning-mode:
+          $ref: '#/components/schemas/PlanningMode'
         config:
           type: object
```

Comment:
Looks like from the sync meeting, we are saying the planning mode should be in config, something like:

```
config:
  type: object
  properties:
     planning-mode:
       $ref: '#/components/schemas/PlanningMode'
  additionalProperties:
    type: string
```

Is this the right understanding?